src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -31,7 +32,7 @@ internal class RazorLanguageServer : AbstractLanguageServer<RazorRequestContext>
 
     public RazorLanguageServer(
         JsonRpc jsonRpc,
-        ILspLogger logger,
+        LoggerWrapper logger,
```

Comment:
Does this need to typed as a `LoggerWrapper` here? Since this this class serves as an adapter to bridge between `ILspLogger` and `ILogger`, it feels like it's a bit implementation-y to include in signatures.

Reply:
I was doing this to make its registration as both an ILspLogger and ILogger easier, but I like your suggestion bellow better!